challenges/intermediate/crack-caesar-cipher/solution/fdb86/rwFiles.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+exports.__esModule = true;
+var hack_caesar_1 = require("./hack-caesar");
```

Comment:
Use `const` instead of `var` for imports

Reply:
sorry this is the wrong file automatically generated by tsc, now I have deleted this file and pushed the correct file: rwFiles.ts